frontend/public/locales/en/translation.json

```diff
@@ -734,6 +734,7 @@
   "clusterSetBinding.edit.select.placeholder": "Select namespaces",
   "clusterSetBinding.edit.title": "Edit namespace bindings",
   "clusterSets": "Cluster sets",
+  "ClusterSets failed to fetch. Check the Gitopscluster and Placement YAML for status errors.": "ClusterSets failed to fetch. Check the Gitopscluster and Placement YAML for status errors.",
```

Comment:
@chenz4027 Did you have someone from doc review this string and is`Gitopscluster` the correct casing?

Reply:
Thanks Kevin, nice catch. It should be GitOpsCluster instead as that's the correct kind for the resource.

Reply:
Will open a separate PR for it